SystemContractsHashes.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Maybe expand this PR to also cover hashes of all system contracts?

Reply:
other contracts added